content/docs/user-guide/packaging/project-export/_index.md

```diff
@@ -11,7 +11,7 @@ The topics in this section provide information about how to use the **Project Ex
 
 | Topic | Description |
 | - | - |
-| [Project Export for Windows and Linux](project-export-cli) | Learn how to use the Project Export CLI to automate preparing your project for release on Windows and Linux. |
-| [Project Export for Android](project-export-android) |  Learn how to use the Project Export CLI to automate preparing your project for release on Android. |
-| [Project Export for iOS](project-export-ios) | (Experimental) Learn how to use the Project Export CLI to automate preparing your project for release on iOS. |
+| [Project Export for Windows and Linux](project-export-pc) | Learn how to use the Project Export functionality to automate preparing your project for release on Windows and Linux. |
```

Comment:
`tool` instead of `functionality` makes more sense here. If you say `functionality`, then you should describe both the UI and the command line version as well.